### PR TITLE
Examples refresh

### PR DESCRIPTION
**Pull request by andressbarajas**
Added press start to exit to some examples.
Removed -n from pvr examples to make it easier to profile

### CODE REVIEW

File: examples/dreamcast/kgl/basic/elements/gl-elements.c [hunk 1]
```diff
@@ -78,6 +78,22 @@ void RenderCallback(GLuint texID) {
     glDisable(GL_KOS_NEARZ_CLIPPING);
 }
 
+int check_start(void) {
```
Comment (gyrovorbis) [inline examples/dreamcast/kgl/basic/elements/gl-elements.c:81]:
```suggestion
static int check_start(void) {
```
Making these `static`, cuz they are never used from outside of this source file/translation unit.

File: examples/dreamcast/kgl/basic/zclip_arrays/gl-arrays.c [hunk 2]
```diff
@@ -86,6 +86,22 @@ void RenderCallback(GLuint texID) {
     glDisable(GL_KOS_NEARZ_CLIPPING);
 }
 
+int check_start(void) {
```
Comment (gyrovorbis) [inline examples/dreamcast/kgl/basic/zclip_arrays/gl-arrays.c:89]:
```suggestion
static int check_start(void) {
```

File: examples/dreamcast/kgl/demos/blur/radial_blur.c [hunk 3]
```diff
@@ -290,10 +290,25 @@ int InputCallback(void) {
     return 1;
 }
 
+int check_start(void) {
```
Comment (gyrovorbis) [inline examples/dreamcast/kgl/demos/blur/radial_blur.c:293]:
```suggestion
static int check_start(void) {
```

File: examples/dreamcast/kgl/demos/multitexture-arrays/gl-multitex-arrays.c [hunk 4]
```diff
@@ -90,6 +90,22 @@ void RenderCallback(GLuint texID0, GLuint texID1) {
     glDisableClientState(GL_VERTEX_ARRAY);
 }
 
+int check_start(void) {
```
Comment (gyrovorbis) [inline examples/dreamcast/kgl/demos/multitexture-arrays/gl-multitex-arrays.c:93]:
```suggestion
static int check_start(void) {
```

File: examples/dreamcast/kgl/demos/multitexture-elements/gl-multitex-elements.c [hunk 5]
```diff
@@ -93,6 +93,22 @@ void RenderCallback(GLuint texID0, GLuint texID1) {
     glDisableClientState(GL_VERTEX_ARRAY);
 }
 
+int check_start(void) {
```
Comment (gyrovorbis) [inline examples/dreamcast/kgl/demos/multitexture-elements/gl-multitex-elements.c:96]:
```suggestion
static int check_start(void) {
```

**Review by gyrovorbis — APPROVED** [review]
Good work, didn't even realize so many still had no way to exit...

**Review by ljsebald — APPROVED** [review]

**Pull request merged**